src/views/account/Homeworks/Document.tsx

```diff
@@ -164,9 +185,255 @@ const HomeworksDocument: Screen<"HomeworksDocument"> = ({ route }) => {
+            <PapillonPicker
+              animated
+              direction="right"
+              delay={0}
+              data={[
+                {
+                  icon: <PencilLine />,
+                  label: "Modifier le devoir",
+                  onPress: () => setShowCreateHomework(true),
+                },
+                {
+                  icon: <Trash2 />,
+                  label: "Supprimer le devoir",
+                  onPress: () => {
+                    Alert.alert(
+                      "Supprimer le devoir",
+                      "Veux-tu vraiment supprimer ce devoir ?",
+                      [
+                        {
+                          text: "Annuler",
+                          isPreferred: true,
+                        },
+                        {
+                          text: "Continuer",
+                          style: "destructive",
+                          onPress: () => {
+                            useAccounts.getState().removeHomework(account.localID, homework.id);
+                            navigation.goBack();
+                          }
+                        }
+                      ]
+                    );
+                  },
+                },
+              ]}
+            >
+              <TouchableOpacity>
+                <MoreHorizontal size={24} color={theme.colors.text} />
+              </TouchableOpacity>
+            </PapillonPicker>
+          )}
         </View>
       </PapillonModernHeader>
 
+      <BottomSheet
+        key={idHomework}
+        opened={showCreateHomework}
+        setOpened={(bool: boolean) => {
+          setShowCreateHomework(bool);
+          if (!bool) {
+            setSelectedPretty(Object.entries(localSubjects || {})[0]?.[1] ?? null);
+            setIdHomework(NaN);
+            setContentHomework(null);
+            setContentHomework(null);
+            setDateHomework(Date.now());
+          }
+          setHomework(homework);
+        }}
+        contentContainerStyle={{
+          paddingHorizontal: 16,
+          borderColor: theme.colors.border,
+          borderWidth: 1,
+        }}
+      >
+        <View style={{ alignSelf: "center", marginTop: 15, flexDirection: "row", gap: 16 }}>
+          <MemoizedNativeText variant="titleLarge" numberOfLines={2}>
+            Modifier un devoir
+          </MemoizedNativeText>
+        </View>
+
+        <MemoizedNativeList style={{ marginTop: 16 }}>
+          <MemoizedNativeItem>
+            <MemoizedNativeText variant="subtitle" numberOfLines={1}>
+              Aperçu
+            </MemoizedNativeText>
+            <View style={{ marginHorizontal: -20, marginTop: 5, marginBottom: -10 }}>
+              <HomeworkItem
+                homework={{
+                  ...homework,
+                  subject: selectedPretty.pretty,
+                  color: selectedPretty.color,
+                  content: contentHomework ?? "Écris le contenu du devoir juste en-dessous :)",
```

Comment:
```suggestion
                  content: contentHomework ?? "Écris le contenu du devoir juste en-dessous",
```
Peut-être un peu de trop le ":)"